app/code/Magento/Captcha/Test/Unit/Controller/Refresh/IndexTest.php

```diff
@@ -62,24 +67,37 @@ protected function setUp()
         $this->viewMock = $this->getMock(\Magento\Framework\App\ViewInterface::class);
         $this->layoutMock = $this->getMock(\Magento\Framework\View\LayoutInterface::class);
         $this->flagMock = $this->getMock(\Magento\Framework\App\ActionFlag::class, [], [], '', false);
+        $this->serializerMock = $this->getMock(
+            \Magento\Framework\Serialize\SerializerInterface::class,
```

Comment:
Please use `Magento\Framework\Serialize\Serializer\Json` instead of `SerializerInterface`.